app/Console/Command/EventShell.php

```diff
@@ -263,6 +263,50 @@ public function cachenids() {
 		$this->Job->saveField('date_modified', date("y-m-d H:i:s"));
 	}
 
+	public function cachebro()
+	{
+		$broHeader = "#fields indicator\tindicator_type\tmeta.source\tmeta.url\tmeta.do_notice\tmeta.if_in\n";
+		$userId = $this->args[0];
+		$user = $this->User->getAuthUser($userId);
+		$id = $this->args[1];
+		$this->Job->id = $id;
+		$format = $this->args[2];
+		$this->Job->saveField('progress', 1);
+		//$types = array_keys($this->Attribute->typeDefinitions);
+		$types = array('ip', 'email', 'domain', 'filename', 'filehash', 'certhash', 'software', 'url'); //Bro types
+		$typeCount = count($types);
+		$dir = new Folder(APP . DS . '/tmp/cached_exports/' . $format, true, 0750);
+
+		if ($user['Role']['perm_site_admin']) {
+			$zipname = DS . 'misp.bro.ADMIN.intel.zip';
+		} else {
+			$zipname = DS . 'misp.bro.' . $user['Organisation']['name'] . '.intel.zip';
+		}
+		$zip = new ZipArchive();
```

Comment:
We have to watch out with this. ZipArchive won't be installed for a lot of installations, unless I am mistaken.